### PR TITLE
build the version string at compile time

### PR DESCRIPTION
**Pull request by camthesaxman**

### CODE REVIEW

File: src/openrct2/Version.h [hunk 1]
```diff
@@ -61,21 +61,12 @@ extern "C"
 {
 #endif
     // The following constants are for automated build servers
-    extern const char *gBuildNumber;
-    extern const char *gBuildServer;
-    extern const char *gGitBranch;
-    extern const char *gCommitSha1;
-    extern const char *gCommitSha1Short;
+    extern const char gBuildNumber[];
```
Comment (janisozaur) [inline src/openrct2/Version.h]: @IntelOrca while touching this, can we get rid of this variable?

**Comment by camthesaxman** [comment]
That replaces Version::GetInfo(). The code functions as before, but instead of  calling a function, it simply reads the constant string gVersionInfoFull.

File: src/openrct2/Version.h (hunk 1 above)
Comment (IntelOrca) [inline src/openrct2/Version.h]: yes

Comment (janisozaur) [inline src/openrct2/Version.h]: @camthesaxman can you remove this variable?

**Comment by camthesaxman** [comment]
Should I remove Version.cpp and just #define the version string in Version.h?

**Comment by janisozaur** [comment]
No. It is specifically done as a variable so that it gets stored in the dumps.

**Pull request merged**